src/core.ts

```diff
@@ -15,7 +15,10 @@ function toBase64 (s: Buffer | string) {
   else return s.toString('base64') // assume a buffer
 }
 
-function each (objOrArr: Record<string, any> | Array<any>, iter: any) {
+function each<T> (
+  objOrArr: Record<string, T> | Array<T>,
+  iter: (t: T, k: string | number, o: Record<string, T> | Array<T>) => void
+) {
```

Comment:
While `iter` was `any` before, here we specify that it's a function that looks like `(value, key, wholeObject) => void`, where `void` means "returns nothing".